wandb/sdk/data_types/utils.py

```diff
@@ -188,8 +188,10 @@ def _log_table_artifact(val: "Media", key: str, run: "LocalRun") -> None:
     sanitized_key = re.sub(r"[^a-zA-Z0-9_\-.]+", "", key)
 
     if isinstance(val, wandb.Table) and val.log_mode == "INCREMENTAL":
+        if run.resumed:
+            incremental_table_util.handle_resumed_run(val, run, key)
```

Comment:
It's not clear what this does without looking at the definition of `handle_resumed_run`; it should really be named better. Also, the helper in `incremental_table_util` and the method on the table are named the exact same, which adds to the confusion. Why do we need a helper function instead of just doing this in the Table method?